classes/system/heka/metric_collector/single.yml

```diff
@@ -14,3 +17,4 @@ parameters:
           database: ${_param:influxdb_database}
           username: ${_param:influxdb_user}
           password: ${_param:influxdb_password}
+          time_precision: ${_param:influxdb_time_precision}
```

Comment:
IIUC you don't need this line. See https://github.com/tcpcloud/salt-formula-heka/pull/17/files#diff-f770008ab30e01fe14f5488b7d7c4f68R82